File: staircase.cpp
#include <iostream>

using namespace std;

void staircase(int n) {
	for(int i=0; i<n; i++) {
		for(int k=0; k<n; k++) {
			if(k<=i) {
				cout << "#";
			}
		}
		cout << "\n";
	}
}

int main()
{
	cout << "Enter number of steps: ";
	int num;
	cin >> num;
	staircase(num);

}

File: staircase.py
import sys

class Solution(object):
	def staircase(self,n):
		
		for i in range(0,n):
			for j in range(0,n):
				if j <= i :
					sys.stdout.write("#")
			sys.stdout.write("\n")
def main():
	num = input("Enter number of steps: ")
	sol = Solution()
	sol.staircase(num)

main()
